src/App.js

```diff
@@ -1,25 +1,53 @@
-import logo from './logo.svg';
-import './App.css';
+import { Switch, Route } from "react-router-dom";
+import React, { useState } from "react";
+import "./App.css";
+import Home from "./Home";
+
+const App = () => {
+  const [contacts, setContact] = useState([]);
```

Comment:
`setContacts` makes more sense since it's a function that updates an array of contacts